lib/reversetunnel/agent.go

```diff
@@ -455,6 +455,18 @@ func (a *agent) handleGlobalRequests(ctx context.Context, requests <-chan *ssh.R
 					a.log.Debugf("Failed to reply to %v request: %v.", r.Type, err)
 					continue
 				}
+			case reconnectRequest:
+				a.log.Debugf("Receieved reconnect advisory request from proxy.")
+				if r.WantReply {
+					err := a.client.Reply(r, true, nil)
+					if err != nil {
+						return trace.Wrap(err)
```

Comment:
My thinking was that this would only fail if there was an issue with the underlying connection. Not certain that is the case but I don't see how else it would fail. But to stay consistent with how the other global requests are handled we can log and wait for the context to be cancelled.